CODEOWNERS

```diff
@@ -5,13 +5,13 @@
 /docs                   @chipzoller
 /pkg/autogen            @eddycharly
 /pkg/background         @realshuting
-/pkg/engine             @marcelmue @realshuting
+/pkg/engine             @marcelmue @realshuting @MariamFahmy98
```

Comment:
I don't think this will work right now as you require write access to the repository (being a member of the kyverno github org) to be added to CODEOWNERS.

See also for reference: https://github.com/renovatebot/renovate/issues/7927#issuecomment-1308613570

Reply:
Once she is added, we will add her as an org member per the contributors guidelines.